### DISCUSSION TITLE
Behanghöhe von HmIP-FROLL verwenden

### DISCUSSION THREAD

**Discussion by jaal2001** [SukramJ/aiohomematic#1555] [General]
Hallo zusammen,
ich habe mich mit den verschiedensten Suchbegriffen durchs Forum gelesen und bin mir trotzdem nicht sicher, ob es möglich ist oder nicht.
Ich möchte für eine Automatisierung (konkret eine Notification) die Behanghöhe von meinen HmIP-FROLL abfragen. Ich brauche nicht den Zu/Offen status, sondern einen konkreten % Schwellwert.
In der HA Steuerung kann ich zwar die aktuelle Höhe wie in der HmIP Zentrale sehen, allerdings finde ich keine Entität, die ich abfragen kann. Etliche Beiträge von SukramJ klingen für mich danach, dass eine viele Entitäten nicht erzeugt werden und ich habe daher die Vermutung, dass die Behanghöhe auch dazugehört.

Gibt es noch eine andere, einfache, Möglichkeit die Behanghöhe abzufragen oder muss ich dazu selbst etwas anlegen? Wenn ich selbst etwas anlegen muss, würde ich mich über einen Hinweis in die richtige Richtung freuen ;)
Danke & Gruß
Jan

**Comment by SukramJ** [comment]
Wird in der 1.62.0 bereitgestellt.
